index.src.html

```diff
@@ -839,6 +914,26 @@ <h4 algorithm id="collectfromcredentialstore-passwordcredential">
             [=current settings object=]'s [=environment settings object/origin=].
   </ol>
 
+  <h4 algorithm id="create-passwordcredential">
+    `PasswordCredential`'s `[[Create]](data)`
+  </h4>
+
+  <dfn for="PasswordCredential" method>\[[Create]](data)</dfn> is called with a
+  {{CredentialCreationData}} (|data|), and returns a {{PasswordCredential}} if one can be created,
```

Comment:
Added. I also ended up dropping the synchronous constructors entirely. Might as well go all in.